src/components/shell/shell.js

```diff
@@ -412,7 +412,6 @@ class Shell extends Component {
         className={playerClasses}
         onTouchEnd={e => this.onTouchEnd(e)}
         onMouseUp={() => this.onMouseUp()}
-        onMouseOver={() => this.onMouseOver()}
         onMouseMove={() => this.onMouseMove()}
```

Comment:
mouse move doesn't handle nav state, it means on keyboard nav it'll stay open -> it handled on `onMouseOver `